### PR TITLE
bt-52: Create header

### LINKED ISSUE #52: Create Header
Image

### PR DESCRIPTION
**Pull request by savinaDaria**
Ticket : #52 
- add header component
- add styles
- add color variables

-Usage:
image
image
image

### CODE REVIEW

File: frontend/src/assets/css/variables.scss [hunk 1]
```diff
@@ -13,4 +13,9 @@
     --checkbox-disabled: #f7f8fc;
     --checkbox-checked: #274f8d;
     --checkbox-disabled-checked: #d5dce8;
+    --header-background: #ffffff;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/variables.scss]: you don't need to duplicate color. You can create just 1 variable for white color

File: frontend/src/bundles/common/components/header/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    border-radius: 5px;
+}
+
+.status.online {
+    background-color: var(--status-online);
+}
+
+.status.offline {
+    background-color: var(--status-offline);
+}
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/header/styles.module.scss]:
```suggestion
.status {
    position: absolute;
    top: 30px;
    left: 30px;
    z-index: 1;
    width: 10px;
    height: 10px;
    border: 2px solid var(--status-border);
    border-radius: 5px;
    
    &.online {
        background-color: var(--status-online);
    }
    &.offline {
        background-color: var(--status-offline);
    }
}
```

File: frontend/src/bundles/common/components/header/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.avatar img {
+    width: 40px;
+    height: 40px;
+    border-radius: 200px;
+}
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/header/styles.module.scss]:
```suggestion
.avatar {
    position: relative;
    right: 15px;
    display: flex;
    align-items: center;
    cursor: pointer;
    
    & img {
        width: 40px;
        height: 40px;
        border-radius: 200px;
    }
}
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/header/styles.module.scss (hunk 2 above)
Comment (savinaDaria) [reply frontend/src/bundles/common/components/header/styles.module.scss]: @nikita-remeslov Earlier there was eslint rule with max depth 0 for styles. As I understand it is allowed now to have depth 1, 
 is it?

**Review by anton-otroshchenko — APPROVED** [review]

**Review by mskmee — APPROVED** [review]
Looks fine as for me

File: frontend/src/bundles/common/components/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+const Header: React.FC<Properties> = ({ avatarUrl, isOnline }) => {
+    return (
+        <header className={styles.header}>
+            <div className={styles.avatar}>
+                <img src={avatarUrl} alt="User Avatar" />
+                <div
+                    className={`${styles.status} ${
+                        isOnline ? styles.online : styles.offline
+                    }`}
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/common/components/header/header.tsx]: Use getValidClassNames to combine styles

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/header/styles.module.scss (hunk 2 above)
Comment (nikita-remeslov) [reply frontend/src/bundles/common/components/header/styles.module.scss]: Yes, I talked about it on our daily

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**